doc/fluidsynth.1

```diff
@@ -221,6 +221,9 @@ Must always to be true for usage by fluidsynth executable.
 .TP
 .B synth.verbose            BOOL  [def=False]
 Print received MIDI events to stdout.
+.TP
+.B synth.volenv             STR   [def='emu' vals:'compliant', 'emu']
```

Comment:
Would it make sense to have the previous behaviour as the default (compliant)?

Reply:
Well... I'm not sure. [Element suggested "emu" to be the default](https://github.com/FluidSynth/fluidsynth/issues/70#issue-238323938) and I'm also not feeling comfortable changing this again to the previous behaviour. So I think we should leave it as it is.